docs/rfcs/1013-parsec-account.md

```diff
@@ -320,31 +320,19 @@ The Parse Account relies on email address validation for multiple operations:
 
 For all those cases, the logic is as follows:
 
-1. The Parsec client sends an initial API request to the server.
-2. The server sends a confirmation email to the user containing a link with a validation token.
-3. The user clicks on the link in the email, which open the Parsec client.
-4. The Parsec client sends a second API request with the validation token to the server.
-5. The server proceed with the actual operation.
+1. The user select the operation from his Parsec client.
+2. The Parsec client sends an initial API request to the server.
+3. The server sends a confirmation email to the user containing a validation code.
+4. The user checks his email to get the validation code, and paste it in his Parsec client.
+5. The Parsec client sends a second API request with the validation code to the server.
+6. The server proceed with the actual operation.
 
-Hence, the need to define a new type of URL link: `ParsecAccountActionAddr`.
+Regarding its format, the validation code is composed of 6 characters using the base32
+alphabet (i.e. numbers + uppercase letters, minus `0/1/8` to avoid confusion with `O/I/B`).
 
-The format is similar to the existing `ParsecActionAddr`, the only difference being
-that `ParsecActionAddr` specifies the organization ID.
-
-`ParsecAccountActionAddr` format: `parsec3://parsec.example.com?a=<action>&p=<payload>`
-
-New `ParsecAccountActionAddr` format can have three variants:
-
-name              | `action` value     | `payload` value
-------------------|--------------------|------------------
-`AccountCreation` | `account_create`   | `base64(msgpack(<validation_token>))`
-`AccountRecovery` | `account_recovery` | `base64(msgpack(<validation_token>))`
-`AccountDeletion` | `account_delete`   | `base64(msgpack(<validation_token>))`
-
-> [!NOTE]
-> We should probably rename `ParsecActionAddr` -> `ParsecOrganizationActionAddr`
-> to avoid confusion with this new `ParsecAccountActionAddr` (this is a purely internal
-> code change, so no risk of backward compatibility issue).
+Since this validation code must be copy&pasted by a end user, it is of much lower entropy
+than the tokens are used in `ParsecInvitationAddr`/`ParsecOrganizationBootstrapAddr`. For
+this reason, a validation code can only be attempted three times before becoming invalid.
```

Comment:
For clarification: If a user has legitimately noodle fingers and fails at the validation code step, what's the expected process ?
- they restart the process from scratch after some time after the last attempt ?
- they contact an administrator to be manually unlocked ?
- should their mail be kept in unverified_emails after their 3rd attempt, or moved to a suspicious emails list ?

Reply:
The user only has to restar the process from scratch (i.e. resending a new validation email containing a new code)

I've added a comment to make this more explicit